### ISSUE TITLE
Работа модерации

### ISSUE THREAD

**Issue by f1nik76** [WALTERXO/telegram-grabber#14]
Устанавливаю модерацию в состояние Вкл, посты уходят в канал мимо технического, меняю состояние, перезагружаю бота, посты так же улетают. Меняю состояние без перезагрузки бота, посты все так же мимо технического канала. ИД канала проверил, права боту в канал дал. В чем может быть проблема?

**Comment by Therealwh** [comment]
Я уже сообщал автору что не корректно работает, не перезапускайте бота играйте с кнопкой. У меня работает когда пишет что модерация выключена. А посты летят
 на модерацию

**Comment by WALTERXO** [comment]
@f1nik76 @Therealwh 
Так, сначала. После каждого нового запуска бота требуется заново включать модерацию и перед этим заново нажимать /start (чтобы увидеть верное отображение включена ли модерация).
Вот так должно выглядеть:

![image](https://github.com/WALTERXO/telegram-grabber/assets/91873172/49276fc1-4947-4b37-819b-2b746a2aa81f)

После нажатия на "Модерация: вкл" выпадает уведомление, что модерация включена, а кнопка меняется на "Модерация: выкл" чтобы можно было выключить модерацию:

![image](https://github.com/WALTERXO/telegram-grabber/assets/91873172/e53686b3-effb-497f-a677-54d497d409ac)


Я думал это интуитивно понятно т.к. применяется везде

**Comment by WALTERXO** [comment]
@f1nik76  @Therealwh 
Вы же это имели в виду?
Или проблема другая?

**Comment by f1nik76** [comment]
Смысл кнопки то понятен. Я про то что в каком бы она не была положении, в технический канал ничего не прилетает. Ставил и так и так, бота рестартовал и кнопкой и перезапуском скрипта. Все равно все сообщения сразу улетают в основной канал без модерации. И еще один вопрос чтобы не открывать, как пожелание, сделать опцию на включение выключение автозамены ссылок.

**Comment by Therealwh** [comment]
@f1nik76  А ты прописал тех канал в конфиг? и добавил бота туда ?

**Comment by f1nik76** [comment]
Да, все прописал сразу. Сейчас разобрался. Думаю надо в фак добавить информацию о том что тех канал и модерация работают только при автограббинге, если через кнопку копировать последние сообщения, то все летит сразу в основной канал независимо от настроек
